includes/admin/class-wc-rest-payments-transactions-controller.php

```diff
@@ -154,4 +160,22 @@ static function ( $filter ) {
 			}
 		);
 	}
+
+	/**
+	 * Formats the incoming transaction date as per the blog's timezone.
+	 *
+	 * @param string|null $transaction_date Transaction date to format.
+	 *
+	 * @return string|null The formatted transaction date as per timezone.
+	 */
+	private function format_transaction_date_with_timestamp( $transaction_date = null ) {
```

Comment:
```suggestion
	private function format_transaction_date_with_timestamp( string|null $transaction_date ) {
```

The parameter is always passed, so I don't see the point of having a null default value